test/compile_test.sh

```diff
@@ -290,3 +290,13 @@ testComplile_BuildPropertiesFileWithMServerVersion()
 
   assertContains "SBT should have been installed" "Downloading sbt launcher for" "$(cat ${STD_ERR})"
 }
+
+testComplile_CreatesExportScript()
+{
+  createSbtProject
+
+  compile
+
+  assertEquals 0 "${RETURN}"
```

Comment:
If you're interested the Node buildpack has some utils that make this a little clearer: https://github.com/heroku/heroku-buildpack-nodejs/blob/master/test/utils#L94